setup.py

```diff
@@ -1,22 +1,46 @@
+from setuptools import find_packages
 from setuptools import setup
 
 
 setup(
     name='ok-tools',
     version='0.1',
+    description='Tools to manage Offene Kanäle',
+    long_description=(
+        'Some tools to help the organisation of Offene Kanäle in Germany,'
+        ' especially in Sachsen-Anhalt.'
```

Comment:
Sollte man Sachsen vielleicht noch übersetzen?
```suggestion
        ' especially in Saxony-Anhalt.'
```